rm/rm-server/src/main/java/org/ow2/proactive/resourcemanager/db/RMDBManager.java

```diff
@@ -423,6 +435,7 @@ public void removeNode(final RMNodeData rmNodeData) {
 
         if (rmdbManagerBuffer.canOperateDatabaseSynchronouslyWithNode(rmNodeData)) {
             try {
+                logger.info("Remove node " + rmNodeData.getName() + " in database");
```

Comment:
Maybe we should put this in debug ?

Reply:
As it is a node removed from the database, it seems an important info, and should not occur too often. So no risk to pollute logs.